manifests/params.pp

```diff
@@ -38,8 +38,8 @@
   $framework_config = {
     'framework.server.name'     => $::fqdn,
     'framework.server.hostname' => $::fqdn,
-    'framework.server.port'     => '4440',
```

Comment:
@liamjbennett should we not enable ssl by default then? The only thing I can think of in that sense is if we break up framework-config.